### PR TITLE
modules/understanding-upgrade-channels: Blocked update edges are still supported

### PR DESCRIPTION
**Pull request by wking**
If we supported them at some point, we will continue to support the edges even if they are no longer recommended.

### CODE REVIEW

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: modules/understanding-upgrade-channels.adoc
Comment (LalatenduMohanty) [reply modules/understanding-upgrade-channels.adoc:108]: As discussed with @wking , we will handle this in a separate PR.

Comment (wking) [reply modules/understanding-upgrade-channels.adoc:108]: I'm fine expanding on the criteria that feed blocked-edge decisions, but I'd rather keep that separate from this pivot to supporting updates over blocked edges.  Because the support pivot is what needs @sferich888's sign-off.  Can we punt your rewording to a follow-up PR?

**Comment by jeana-redhat** [comment]
@sferich888 I'm going to label this with 4.5+ to mark it for cherrypicking into all current and future branches. Let me know if any of those should be omitted.

**Comment by sdodson** [comment]
@jeana-redhat @codyhoag What's left to move this out? Given that this is all after QE has signed off on releases (we only ship errata after their sign off) I'm not sure they'll be concerned about this change at all.

**Comment by jiajliu** [comment]
LGTM.

**Comment by jeana-redhat** [comment]
@sdodson thanks for following up. Looks like we thought we needed QE ack, and now we have it even though as you say it's not required. Only outstanding question as far as I know is confirmation that this goes into all releases (4.5+) - do you know if that's right? If so, I'll review for docs compliance and merge/cherrypick.

**Review by jeana-redhat — APPROVED** [review]
docs review: lgtm

**Comment by sdodson** [comment]
@jeana-redhat yes, all currently supported versions is fine

**Comment by jeana-redhat** [comment]
@sdodson ACK and done - tyvm! :+1:

**Pull request merged**
